tagger/train/train.py

```diff
@@ -3,25 +3,25 @@
 
 #Import from other modules
 from tagger.data.tools import make_data, load_data, to_ML
-from tagger.plot.basic import loss_history, basic_ROC, pt_correction_hist, rms
+from tagger.plot.basic import loss_history, basic
 import models
 
 #Third parties
 import numpy as np
 import tensorflow as tf
 import tensorflow_model_optimization as tfmot
+from tensorflow.keras.callbacks import EarlyStopping, ReduceLROnPlateau, ModelCheckpoint
+from sklearn.utils.class_weight import compute_class_weight
 
 # GLOBAL PARAMETERS TO BE DEFINED WHEN TRAINING
+tf.keras.utils.set_random_seed(420) #not a special number 
 BATCH_SIZE = 1024
 EPOCHS = 100
 VALIDATION_SPLIT = 0.1 # 10% of training set will be used for validation set. 
 
 # Sparsity parameters
 I_SPARSITY = 0.0 #Initial sparsity
-F_SPARSITY = 0.6 #Final sparsity
-
-# Loss function parameters
-GAMMA = 0.3 #Loss weight for classification, 1-GAMMA for pt regression
+F_SPARSITY = 0.1 #Final sparsity
```

Comment:
Why is this so much lower than before?

Reply:
I kept this low just to make sure there was no degredation in performance, I'll gradually increase this in the future as I make sure that nothing in the performance changes. Thank you for pointing this out.